drivers/hwinfo/hwinfo_nrf.c

```diff
@@ -9,6 +9,9 @@
 #include <string.h>
 #include <hal/nrf_ficr.h>
 #include <sys/byteorder.h>
+#ifndef CONFIG_BOARD_QEMU_CORTEX_M0
```

Comment:
Is this only for QEMU? Or should it check on a given nRF family instead?

Reply:
QEMU only.